### PR TITLE
Add new attribute to the title

### PR DESCRIPTION
**Pull request by vitorflg**
Hi! It would be great to have a attribute to the title to controls it's size, today it's only possible to change the row title size.

### CODE REVIEW

**Comment by binodswain** [comment]
Can you add before and after screenshot of the component from example folder.

**Comment by vitorflg** [comment]
> Can you add before and after screenshot of the component from example folder.

Sure! 

Before(without passing titleTextSize prop)

Screen Shot 2020-12-29 at 12 10 22 AM

After(passing titleTextSize prop with "48px")

Screen Shot 2020-12-29 at 12 11 07 AM

File: src/styles.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $default-color: black;
 $white: white;
+$default-title-size: 16px;
```
[the remark this answers is not in the document]
Comment (vitorflg) [reply src/styles.scss:3]: @binodswain Looking again here, looks like it's `1.62rem`(`25.92px`), could be it?

**Review by binodswain — APPROVED** [review]
looks good.

**Comment by binodswain** [comment]
react-faq-component@1.2.2

**Pull request merged**